### ISSUE TITLE
Better naming to the machines

### ISSUE THREAD

**Issue by noamha** [scylladb/cassandra-test-and-deploy#51]
It would be great if you would also set the machines name according to the ANSIBLE_EC2_PREFIX and also add an incremental numbering so one can uniquely and easily identify each machines when debugging.

Thanks

**Comment by tzach** [comment]
EC2 do assign an incremental value called AMI launch index, which is visible from EC2 admin.
Is it sufficient, or do you need more than that?

**Comment by tzach** [comment]
> It would be great if you would also set the machines name according to the ANSIBLE_EC2_PREFIX 

Agree. I moved away from that because it makes the Ansible host filtering much more complex.
Might be solvable by a smarter man.
